packages/xo-server/src/xapi/mixins/networking.js

```diff
@@ -59,7 +59,11 @@ export default {
       ],
     },
     lockingMode: {
-      set: (value, vif) => vif.set_locking_mode(value),
+      set: (value, vif) => {
+        if (value !== undefined) {
```

Comment:
This case is properly handled by `makeEditObject`. See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xapi/utils.js#L266